Managers/SaveManager.gd

```diff
@@ -0,0 +1,31 @@
+extends Node
+
+signal start_save
+signal start_load
+
+var config_file: ConfigFile
```

Comment:
should it be called `save_file` instead ?

Reply:
Good idea. I named it config_file since it's the type, but save_file is better readability for us.